core-it-suite/src/test/resources/mng-7045/pom.xml

```diff
@@ -25,8 +25,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.8.1</version>
         <configuration>
-          <source>8</source>
-          <target>8</target>
+          <source>7</source>
```

Comment:
Is that valid? I thought dropping the `1.` is allowed only from 8 onwards, but I'm not sure...

Reply:
Likely depends on the Java version used

Reply:
Good point, I was a bit too hasty there. For my JDK (OpenJDK) it worked with "7", but made it "1.7" just to be sure. It works with that as well.